vms/platformvm/utxo/camino_locked.go

```diff
@@ -955,6 +964,15 @@ func (h *handler) VerifyUnlockDeposit(
 				err,
 			)
 		}
+
+		signers, err := state.GetMultisigUTXOSigners(utxo)
```

Comment:
may be it would be better to return OutputOwners instead of output ? then the replaceUtxoOut func would also simplier, imo